angelspetrochem/prototypes/technology/petrochem-basic-chemistry.lua

```diff
@@ -290,7 +290,11 @@ data:extend(
         {
           type = "unlock-recipe",
           recipe = "coke-purification-2"
-        }
+        },
+	    {
+          type = "unlock-recipe",
+          recipe = "pellet-coke"
+        },

```

Comment:
Small indentation change: use spaces instead of tabs.
```suggestion
        {
          type = "unlock-recipe",
          recipe = "pellet-coke"
        },
```